### DISCUSSION TITLE
Email sending

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#14883] [How to]
Gents, I have set emails as follows:
Mail - Roundcube > imap (gmail )
SMTP Configuration - Amazon SES

Now when I click some email (accounts for instance) the email is not going out at all. 
What would be the correct config in this case?

Best

**Comment by waran70** [comment]
Setting SMTP and save sent email (in SMTP config) should be enough. Mail scanner should have separate SMTP config (or sourced from SMTP list). Here is lack of integrated email composer like in vtiger we can observe (or others). What in many instances happen is that admins dont wanna to give users mailboxes but let em deal with tickets only (ticketing is very efficient email management anyway).

**Comment by apcloic** [comment]
@s2b-git 
Check in Emails to send if you see your emails and also check CRON if Emails sender task is active.
https://gitstable.yetiforce.com/index.php?module=Mail&parent=Settings&view=List
https://gitstable.yetiforce.com/index.php?module=CronTasks&parent=Settings&view=List

**Comment by IanMuller** [comment]
There is a problem with identities.  Normally identity should be sourced from user profile. Here limited user can set it himself and what funny is he can make it default, and all other users identity will be set to this default. 
I was always thinking Roundcube is not the best idea. Better was to (as per above) use internal email composer with native window and smtp server. Inbox: sometimes useful, but can be without using scanner and helpdesk

**Comment by s2b-git** [comment]
User should not have ability to change email, signature, or smtp server
Admin should be able to set it on his side. In large system there might me a lot of mad users, and wrongful actions shall be prevented

Best
